content/en/blog/2019/announcing-1.3/index.md

```diff
@@ -0,0 +1,56 @@
+
+*Container ports are no longer required*: In the past, Istio required that pods explicitly declare the Kubernetes `containerPort` for each container as a security measure against trampolining traffic. Istio 1.3 has a secure and simpler way of handling all inbound traffic on any port into a Pod without requiring the `containerPort` declarations. We have also completely eliminated infinite loops caused in the IPtables rules when pods send traffic to themselves.
+
+*Customize generated Envoy configuration*: While Istio 1.3 focuses on out of box usability, it also caters to the expert users who would like to use advanced features in Envoy that Istio does not expose via its Networking APIs. We have enhanced the `EnvoyFilter` API to allow users to fully customize the Envoy HTTP route configuration returned by RDS, HTTP/TCP listeners and their filter chains returned by LDS, and the set of clusters returned by CDS. You get the best of both worlds - leverage  Istio to integrate with Kubernetes, handle large fleets of Envoys in an efficient manner and still retain the ability to customize the generated Envoy configuration to meet specific requirements within your infrastructure.
+
+*Other enhancements*:
+
+- `istioctl` has gained a plethora of debugging features that will highlight various issues in your mesh installation. Checkout the `istioctl` [reference manual](/docs/reference/commands/istioctl/) for the set of all supported features.
+
+- Locality aware load balancing graduated from experimental to default in this release too. Istio can now take advantage of existing locality information to prioritize load balancing pools and favor sending requests to the closest backends.
+
+- Better support for headless services with Istio mTLS 
+
+- We enhanced control plane monitoring in the following ways:
+  - Added new metrics to monitor configuration state
+  - Added metrics for sidecar injector
+  - Added a new Grafana dashboard for Citadel
+  - Improved the Pilot dashboard to expose additional key metrics
+
+- Added the new [Istio Deployment Models concept](/docs/concepts/deployment-models/) to help you decide what deployment model suits your needs.
+
+- Organized the content in of our [Operations Guide](/docs/ops/) and created a [section with all troubleshooting tasks](/docs/ops/troubleshooting) to help you find the information you seek faster.
+
+Thanks to our community testers, our [documentation](/docs/) is tested for every release including Istio 1.3!
```

Comment:
I'd remove this paragraph. I certainly hope stuff is tested before release, calling it out seems redundant.